### PR TITLE
Add Veracode Actions

### PR DESCRIPTION
**Pull request by mtawadrousv**
### :warning: [Contribution Guidelines](contributing.md) :warning:

Thank you for your contribution! Please ensure you have followed the guidelines provided below before submitting your PR:

#### Code of Conduct

- [X] I have read and agreed to the [Contributor Code of Conduct](code-of-conduct.md).

#### Adding Your Action: Requirements

Ensure you can check all the boxes:

- [X] The organization that holds the action's source repository is a registered member of the Technology Partner Program (TPP). More about TPP [here](http://partner.github.com).
- [X] Code scanning is enabled on the repo holding the action.
- [X] Dependabot is enabled on the repo.
- [X] MFA is enabled for the maintainers of the Action.
- [X] Verified Badge for the organization that owns the Action.
- [X] Security Policy is present in the repo holding the action.

#### Ensure fields are filled correctly:

- [X] **Category**: Selected only ONE category from the list of in the [Contribution Guidelines](contributing.md).
- [X] **Partner**: Name of the organization that is registered with TPP and holds the action's source repository.
- [X] **Action**: Provided a direct hyperlink to your GitHub Action.
- [X] **Description**: Maximum of 100 characters.
- [X] **Security Policy**: Hyperlinked to the security policy in the source repo of the Action.

By submitting this PR, you acknowledge that you have checked off all boxes and followed the [contribution guidelines](contributing.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +18,9 @@ Learn more about GitHub Actions [here](https://github.com/features/actions).
 |Code review|Mergify|[Add Linear author as reviewer](https://github.com/Mergifyio/gha-add-linear-author-as-reviewer)|A GitHub action that request review of the Linear issue author.|[Security Policy](https://github.com/Mergifyio/gha-add-linear-author-as-reviewer/blob/main/SECURITY.md)|
 |Deployment|Pulumi|[Pulumi GitHub Actions](https://github.com/pulumi/actions)|Deploy apps and infrastructure to your cloud of choice, using your favorite language and GitHub.|[Security Policy](https://github.com/pulumi/actions/security/policy)|
 |Security|StepSecurity|[Harden-Runner](https://github.com/step-security/harden-runner)|Runtime security for GitHub Actions across GitHub-hosted and self-hosted runners.|[Security Policy](https://github.com/step-security/harden-runner/blob/main/SECURITY.md)|
+|Security|Veracode|[Upload & Scan](https://github.com/marketplace/actions/veracode-upload-and-scan)|Automatically scan your code using Veracode for first-party vulnerabilities|[Security Policy](https://github.com/veracode/veracode-uploadandscan-action/blob/master/SECURITY.md)|
```
Comment (mattlovestech) [inline README.md:21]: @mtawadrousv do you mind using the repo links instead of the marketplace urls for each of the entries?

**Review by mattlovestech — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mtawadrousv) [reply README.md]: @mattlovestech Sorry about that - just updated

Comment (mtawadrousv) [reply README.md]: Hey @mattlovestech - just resolved the conflicts, let me know if you need anything else

**Review by mattlovestech — DISMISSED** [review]

**Comment by mattlovestech** [comment]
@mtawadrousv apologies for the delay - looks like there is a conflict now but, I approved on my end.

**Comment by mtawadrousv** [comment]
Should be resolved now

**Pull request merged**